### PR TITLE
Photo input – support for multiple photos

### PR DESCRIPTION
**Pull request by sudhacheran**
- Enabled support for uploading multiple photos
- Added the option to delete uploaded photos.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "HTTPS=true node scripts/start.js",
```
Comment (markborkum) [inline package.json:112]: If the protocol for the host is changed from HTTP to HTTPS, and local storage is keyed on the URL, does this mean that users will lose access to their stored projects that were created prior to this change?

File: src/components/photo.tsx [hunk 2]
```diff
@@ -1,85 +1,105 @@
...
 import GpsCoordStr from './gps_coord_str'
 import type PhotoMetadata from '../types/photo_metadata.type'
 
 interface PhotoProps {
     description: React.ReactNode
-    id: string
     label: string
-    metadata: PhotoMetadata
-    photo: Blob | undefined
+    photos: { id: string; photo: Blob; metadata: PhotoMetadata }[] // Array of photo objects with metadata
```
Comment (markborkum) [inline src/components/photo.tsx:12]: What happens to preexisting projects from before this change, where the `PhotoProps.photo` attribute is defined?

File: package.json (hunk 1 above)
Comment (sudhacheran) [reply package.json:112]: "start" script is used to launch the application locally (localhost). "build" script is used in deploying the app in test / prod servers.  I'll revert this change.

File: src/components/photo.tsx (hunk 2 above)
Comment (sudhacheran) [reply src/components/photo.tsx:12]: Thanks for pointing this out. I will work on the fix for this scenario.

Comment (sudhacheran) [reply src/components/photo.tsx:12]: I have tested this feature with the photos stored before this change. Existing photo will be populated as one photo with the option to delete and 'Add Photo' button.  
PhotoProps.photo is now removed and added as PhotoProps.photos to hold multiple photos and the respective metadata. It is updated accordingly in the wrapper class - https://github.com/pnnl/Quality-Install-Tool/blob/17-improve-photoinput-component/src/components/photo_wrapper.tsx#L132

**Review by markborkum — APPROVED** [review]
Looks good to me. Thanks, @sudhacheran, for addressing the comments.

**Pull request merged**